backend/src/test/java/ai/verta/modeldb/ExperimentRunTest.java

```diff
@@ -9879,7 +9874,8 @@ public void findExperimentRunsHyperparameterWithSortKeyRepository()
         3,
         response.getExperimentRunsCount());
 
-    hyperparameterFilter = Value.newBuilder().setStringValue("abc").build();
+    // TODO: Currently string value not supported in metrics / hyperparameters
```

Comment:
the todo is incorrect , we do not support mismatch of datatypes.
We do support if all hyperparameters are string

Reply:
there should be a test case for filtering on a string hyperpparameter is one does not exist already